TestAssets/TestProjects/AllResourcesInSatellite/AllResourcesInSatellite.csproj

```diff
@@ -1,11 +1,31 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
+
   <PropertyGroup>
     <OutputType>Exe</OutputType>
+    <AssemblyOriginatorKeyFile>TestPublicKeyOnly.snk</AssemblyOriginatorKeyFile>
+    <SignAssembly>true</SignAssembly>
+    <PublicSign>true</PublicSign>
+  </PropertyGroup>
+
+  <!-- 
+    Switch target based on current OS to get more test coverage:
```

Comment:
Wouldn't something like the following make more sense?

```xml
<TargetFrameworks>netcoreapp1.0</TargetFrameworks>
<TargetFrameworks Condition="'$(OS)' == 'Windows_NT'">$(TargetFrameworks);net46</TargetFrameworks>
<RuntimeIdentifier Condition="'$(TargetFramework)' == 'net46'">win7-x86</RuntimeIdentifier>
```

That way when you run the tests locally on Windows, you get coverage of both scenarios.